src/collective/cover/interfaces.py

```diff
@@ -30,14 +32,8 @@ def getSearchResults(self, filter_portal_types, rooted, document_base_url, searc
 
 
 class ICoverUIDsProvider(Interface):
-
-    def getUIDs(self):
-        """Get UUIDs associated with the object.
-            could be the UUID of the object or a
-            list of related UUIDs.
-
-        @return: iterable of UUIDs
-        """
+    """GenericUIDsProvider interface will be removed in collective.cover v1.7."""
+    warnings.warn(__doc__, DeprecationWarning)
```

Comment:
No problem, we know that it's not possible to check for every possible corner case. If you agree with the PR do you think it's possible to make another release right after it's merging?